### DISCUSSION TITLE
Please contact us regarding language settings.

### DISCUSSION THREAD

**Discussion by DeokUnBae** [Mr-MIBonk/M.I.B._More-Incredible-Bash#72] [Q&A]
Hello, I always appreciate your material. Looking at the source, it seems that language setting is possible in the setlang part. If you don't mind, can you guide me on how to migrate the KR language?
Thank you for always. Answers I'll wait. My vehicle is a 2016 Audi A6.

**Comment by harman-f** [comment]
What do you mean by "migrate"?

Which FW is you unit currently running?
